gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Feb 22 17:53:41 JST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1-bin.zip
```

Comment:
> Always use the ./gradlew wrapper ... command when updating Gradle

I tried this, and it did not work. I wasted a bit of time on it, and made a judgement call to manually move forward. My reasoning is based on discussions we had earlier: this project is not currently in use, and the rules will be deleted. It is essentially a _new_ project, and as such, it is entirely appropriate, imo, to not worry about "migrating" in the traditional sense, and rather, advance fully to the most up-to-date API versions. The only caveat being that, as also discussed, we are using the existing rules to validate that this is working correctly with the `lintChecks` function.

> Use the Gradle [8.2.1](https://docs.gradle.org/8.2.1/release-notes.html), that is, instead of [8.1](https://docs.gradle.org/8.1/release-notes.html),

I agree, as mentioned above, I think we should be move to the latest available versions. 👍

Reply:
Resolved in: https://github.com/wordpress-mobile/WordPress-Lint-Android/pull/10/commits/c205c48b4cde5c4c0d2950b4d96bd3ae49a2da82

Reply:
Awesome, thanks for this update @mkevins ! 💯

Minor (🔍): I tried running `./gradlew wrapper --gradle-version=8.2.1 --distribution-type=all` again on my side and I still see a couple of updates needing to happen, I wonder why those didn't happen on your side already, you used this command, right? 🤔 

1. `gradle-wrapper.properties` -> `validateDistributionUrl=true`
2. gradlew -> `which java`

Can you please try again? 🙏

Reply:
Resolved in: https://github.com/wordpress-mobile/WordPress-Lint-Android/pull/10/commits/c1a6c32e9f82567e581b019a53970dda6406ee96

Reply:
Yea, I am not sure what's Gradle problem lately, but I too found that running it twice is the safest way, especially if you don't see the `.jar` updated after running it once. 🤷 

Thank YOU! 🥇

Reply:
It seems that this is expected - I just forgot to run it a second time. But, it should be _safe_, even if the wrapper is left out-of-date:

> Note that running the wrapper task once will update gradle-wrapper.properties only, but leave the wrapper itself in gradle-wrapper.jar untouched. This is usually fine as new versions of Gradle can be run even with ancient wrapper files. If you nevertheless want all the wrapper files to be completely up-to-date, you’ll need to run the wrapper task a second time.

[source](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper)

That said, I prefer to keep it _all_ up to date, so I'm glad you raised it. 👍

Reply:
Ow, this is a great find @mkevins , thanks for looking into this! 🔬

PS: I am pretty sure this is a new Gradle thingy as this only started happening quite recently.